src/style.css

```diff
@@ -1,3 +1,92 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial, Helvetica;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  /* width: 100%; */
+  margin-bottom: 20px;
+  width: 1024px;
+}
+
+.logo_png {
+  display: block;
+  width: 40px;
+  height: 40px;
+
+  /* position: relative; */
+
+  /* top: 10px; */
+}
+
+a.blue {
+  color: var(--main-color);
+}
+
+.container {
+  display: flex;
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: 20px;
+  margin-top: 0;
+  list-style: none;
+  justify-content: center;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  justify-content: center;
+  margin-right: 20px;
```

Comment:
the last item shouldn't have margin-right (read about :last-child) - it's said in description